---
layout: default
title: Publications
description: 'Publications of our Lab.'
---
<blockquote>
<h5>COPYRIGHT NOTICE</h5>
<h5>Some documents distributed by this site have been provided by the contributing authors as a means to ensure timely dissemination of scholarly and technical work on a noncommercial basis. Copyright and all rights therein are maintained by the authors or by other copyright holders, notwithstanding that they have offered their works here electronically. It is understood that all persons copying this information will adhere to the terms and constraints invoked by each author&#8217;s copyright. These works may not be reposted without the explicit permission of the copyright holder.</h5>
</blockquote>
<p>This folder is a repository for all papers. For specific paper you can use filter below:</p>
<div id="bibtex_sw"><a name="tppubs" id="tppubs"></a><div class="teachpress_cloud"></div><div class="teachpress_filter"><select name="yr" id="yr" onchange="teachpress_jumpMenu('parent',this,0)">
          <option value="/?page_id=63&amp;tgid=&amp;type=&amp;auth=#tppubs">All years</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2017&amp;type=&amp;auth=#tppubs" >2017</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2016&amp;type=&amp;auth=#tppubs" >2016</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2015&amp;type=&amp;auth=#tppubs" >2015</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2014&amp;type=&amp;auth=#tppubs" >2014</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2013&amp;type=&amp;auth=#tppubs" >2013</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2012&amp;type=&amp;auth=#tppubs" >2012</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2011&amp;type=&amp;auth=#tppubs" >2011</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2010&amp;type=&amp;auth=#tppubs" >2010</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2009&amp;type=&amp;auth=#tppubs" >2009</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2008&amp;type=&amp;auth=#tppubs" >2008</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2007&amp;type=&amp;auth=#tppubs" >2007</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2006&amp;type=&amp;auth=#tppubs" >2006</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2005&amp;type=&amp;auth=#tppubs" >2005</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2004&amp;type=&amp;auth=#tppubs" >2004</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2003&amp;type=&amp;auth=#tppubs" >2003</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2002&amp;type=&amp;auth=#tppubs" >2002</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2001&amp;type=&amp;auth=#tppubs" >2001</option><option value = "/?page_id=63&amp;tgid=&amp;yr=2000&amp;type=&amp;auth=#tppubs" >2000</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1999&amp;type=&amp;auth=#tppubs" >1999</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1998&amp;type=&amp;auth=#tppubs" >1998</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1997&amp;type=&amp;auth=#tppubs" >1997</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1996&amp;type=&amp;auth=#tppubs" >1996</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1995&amp;type=&amp;auth=#tppubs" >1995</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1994&amp;type=&amp;auth=#tppubs" >1994</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1993&amp;type=&amp;auth=#tppubs" >1993</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1992&amp;type=&amp;auth=#tppubs" >1992</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1991&amp;type=&amp;auth=#tppubs" >1991</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1989&amp;type=&amp;auth=#tppubs" >1989</option><option value = "/?page_id=63&amp;tgid=&amp;yr=1988&amp;type=&amp;auth=#tppubs" >1988</option></select><span style="padding-left:10px; padding-right:10px;"><select name="type" id="type" onchange="teachpress_jumpMenu('parent',this,0)">
                   <option value="/?page_id=63&amp;tgid=&amp;yr=&amp;auth=#tppubs">All types</option>
                         <option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=article&amp;auth=#tppubs" >Articles</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=collection&amp;auth=#tppubs" >Collections</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=conference&amp;auth=#tppubs" >Conferences</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=inbook&amp;auth=#tppubs" >Inbooks</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=mastersthesis&amp;auth=#tppubs" >Masterstheses</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=phdthesis&amp;auth=#tppubs" >PhD Theses</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=techreport&amp;auth=#tppubs" >Techreports</option>
                 </select></span><select name="pub-author" id="pub-author" onchange="teachpress_jumpMenu('parent',this,0)">
                  <option value="/?page_id=63&amp;tgid=&amp;yr=&amp;type=#tppubs">All authors</option>
                         <option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=&amp;auth=9#tppubs" >Eduardo Miranda</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=&amp;auth=2#tppubs" >LIS UNICAMP</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=&amp;auth=8#tppubs" >Lucas Carvalho</option><option value = "/?page_id=63&amp;tgid=&amp;yr=&amp;type=&amp;auth=5#tppubs" >MÃ¡rcio Saraiva</option>
                </select></div><p align="center"></p><div class="tablenav"><div class="tablenav-pages"><span class="displaying-num">440 entries</span> <a class="first-page disabled">&laquo;</a> <a class="prev-page disabled">&lsaquo;</a>  1 of 28 <a href="/?page_id=63&amp;limit=2&amp;" title="next page" class="page-numbers">&rsaquo;</a> <a href="/?page_id=63&amp;limit=28&amp;" title="last page" class="page-numbers">&raquo;</a> </div></div><table class="teachpress_publication_list"><tr><td><h3 class="tp_h3">2017</h3></td></tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Carvalho,  Lucas A. M. C.; Essawy,  Bakinam T.; Garijo,  Daniel; Medeiros,  Claudia Bauzer; Gil,  Yolanda</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('447','tp_links')" style="cursor:pointer;">Requirements for Supporting the Iterative Exploration of Scientific Workflow Variants</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">2017 Workshop on Capturing Scientific Knowledge (SciKnow), held in conjunction with the ACM International Conference on Knowledge Capture (K-CAP), </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_447" class="tp_show" onclick="teachpress_pub_showhide('447','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_447" class="tp_show" onclick="teachpress_pub_showhide('447','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_447" class="tp_show" onclick="teachpress_pub_showhide('447','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=135#tppubs" title="Show all publications which have a relationship to this tag">Knowledge Capture</a>, <a href="/?page_id=63&amp;tgid=99#tppubs" title="Show all publications which have a relationship to this tag">Scientific Workflows</a>, <a href="/?page_id=63&amp;tgid=136#tppubs" title="Show all publications which have a relationship to this tag">Workflow Variants</a>, <a href="/?page_id=63&amp;tgid=134#tppubs" title="Show all publications which have a relationship to this tag">Workshop</a>)</p><div class="tp_bibtex" id="tp_bibtex_447" style="display:none;"><div class="tp_bibtex_entry">@conference{Carvalho2017b,<br />
title = {Requirements for Supporting the Iterative Exploration of Scientific Workflow Variants},<br />
author = {Lucas A. M. C. Carvalho and Bakinam T. Essawy and Daniel Garijo and Claudia Bauzer Medeiros and Yolanda Gil},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2018/01/workflow-variants-sciknow-2017-camera-ready.pdf},<br />
year  = {2017},<br />
date = {2017-12-04},<br />
booktitle = {2017 Workshop on Capturing Scientific Knowledge (SciKnow), held in conjunction with the ACM International Conference on Knowledge Capture (K-CAP)},<br />
pages = {1-8},<br />
abstract = {Workflow systems support scientists in capturing computational experiments and managing their execution.  However, such systems are not designed to help scientists create and track the many related workflows that they build as variants, trying different software implementations and distinct ways to process data and deciding what to do next by looking at previous workflow results.  An initial workflow will be changed to create many new variants thereof that differ from each other in one or more steps. Our goal is to support scientists in the iterative design of computational experiments by assisting them in the creation and management of workflow variants.  In this paper, we present several use cases for creating workflow variants in hydrology, from which we specify requirements for workflow variants.  We also discuss major research directions to address these requirements.},<br />
keywords = {Knowledge Capture, Scientific Workflows, Workflow Variants, Workshop}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('447','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_447" style="display:none;"><div class="tp_abstract_entry">Workflow systems support scientists in capturing computational experiments and managing their execution.  However, such systems are not designed to help scientists create and track the many related workflows that they build as variants, trying different software implementations and distinct ways to process data and deciding what to do next by looking at previous workflow results.  An initial workflow will be changed to create many new variants thereof that differ from each other in one or more steps. Our goal is to support scientists in the iterative design of computational experiments by assisting them in the creation and management of workflow variants.  In this paper, we present several use cases for creating workflow variants in hydrology, from which we specify requirements for workflow variants.  We also discuss major research directions to address these requirements.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('447','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_447" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2018/01/workflow-variants-sciknow-2017-camera-ready.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2018/01/workflow-variants-scikno[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2018/01/workflow-variants-scikno[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('447','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Santo,  Jacqueline Midlej do EspÃ­rito; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('445','tp_links')" style="cursor:pointer;">Semantic Interoperability of Clinical Data</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Lecture Notes in Bioinformatics (LNBI) - Proceedings of 12th International Conference on Data Integration in the Life Sciences, </span><span class="tp_pub_additional_volume">10649 </span, <span class="tp_pub_additional_address">Luxemburgo, Luxemburgo, </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_445" class="tp_show" onclick="teachpress_pub_showhide('445','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_445" class="tp_show" onclick="teachpress_pub_showhide('445','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_445" class="tp_show" onclick="teachpress_pub_showhide('445','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=93#tppubs" title="Show all publications which have a relationship to this tag">Data integration</a>, <a href="/?page_id=63&amp;tgid=133#tppubs" title="Show all publications which have a relationship to this tag">healthcare</a>)</p><div class="tp_bibtex" id="tp_bibtex_445" style="display:none;"><div class="tp_bibtex_entry">@conference{Santo2017,<br />
title = {Semantic Interoperability of Clinical Data},<br />
author = {Jacqueline Midlej do EspÃ­rito Santo and Claudia Bauzer Medeiros},<br />
editor = {Springer International Publishing AG},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/DILS-jacqueline.pdf},<br />
year  = {2017},<br />
date = {2017-11-14},<br />
booktitle = {Lecture Notes in Bioinformatics (LNBI) - Proceedings of 12th International Conference on Data Integration in the Life Sciences},<br />
volume = {10649},<br />
address = {Luxemburgo, Luxemburgo},<br />
abstract = {The interoperability of clinical information systems is particularly complicated due to the use of outdated technologies and the absence of consensus about standards. The literature applies standard-based approaches to achieve clinical data interoperability, but many systems do not adopt any standard, requiring a full redesigning process. Instead, we propose a generic computational approach that combines a hierarchical organization of mediator schemas to support the interoperability across distinct data sources. Second, our work takes advantage of knowledge bases to be linked to clinical data, and exploit these semantic linkages via queries. The paper shows case studies to validate our<br />
proposal.},<br />
keywords = {Data integration, healthcare}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('445','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_445" style="display:none;"><div class="tp_abstract_entry">The interoperability of clinical information systems is particularly complicated due to the use of outdated technologies and the absence of consensus about standards. The literature applies standard-based approaches to achieve clinical data interoperability, but many systems do not adopt any standard, requiring a full redesigning process. Instead, we propose a generic computational approach that combines a hierarchical organization of mediator schemas to support the interoperability across distinct data sources. Second, our work takes advantage of knowledge bases to be linked to clinical data, and exploit these semantic linkages via queries. The paper shows case studies to validate our<br />
proposal.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('445','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_445" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/DILS-jacqueline.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/DILS-jacqueline.pdf" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/DILS-jacqueline.pdf</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('445','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Diaz,  Juan S. B.; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('446','tp_links')" style="cursor:pointer;">WorkflowHunt: combining keyword and semantic search in scientific workflow repositories</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Proceedings of the IEEE 13th International Conference on eScience 2017, </span><span class="tp_pub_additional_publisher">IEEE, </span><span class="tp_pub_additional_address">Auckland, New Zealand, </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_446" class="tp_show" onclick="teachpress_pub_showhide('446','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_446" class="tp_show" onclick="teachpress_pub_showhide('446','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_446" class="tp_show" onclick="teachpress_pub_showhide('446','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=99#tppubs" title="Show all publications which have a relationship to this tag">Scientific Workflows</a>, <a href="/?page_id=63&amp;tgid=25#tppubs" title="Show all publications which have a relationship to this tag">Semantic Annotation</a>, <a href="/?page_id=63&amp;tgid=100#tppubs" title="Show all publications which have a relationship to this tag">Workflow Retrieval</a>)</p><div class="tp_bibtex" id="tp_bibtex_446" style="display:none;"><div class="tp_bibtex_entry">@conference{Diaz2017,<br />
title = {WorkflowHunt: combining keyword and semantic search in scientific workflow repositories},<br />
author = {Juan S. B. Diaz and Claudia Bauzer Medeiros},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/PID4958635.pdf},<br />
year  = {2017},<br />
date = {2017-10-24},<br />
booktitle = {Proceedings of the IEEE 13th International Conference on eScience 2017},<br />
publisher = {IEEE},<br />
address = {Auckland, New Zealand},<br />
abstract = {Scientific datasets, and the experiments that analyze them are growing in size and complexity, and scientists are facing difficulties to share such resources. Some initiatives have emerged to try to solve this problem. One of them involves the use of scientific workflows to represent and enact experiment execution. There is an increasing number of workflows that are potentially relevant for more than one scientific domain. However, it is hard to find workflows suitable for reuse given an experiment. Creating a workflow takes time and resources, and their reuse helps scientists to build new workflows faster and in a more reliable way. Search mechanisms in workflow repositories should provide different options for workflow discovery, but it is difficult for generic repositories to provide multiple mechanisms. This paper presents WorkflowHunt, a hybrid architecture for workflow search and discovery for generic repositories, which combines keyword and semantic search to allow finding relevant workflows using different search methods. We validated our architecture creating a prototype that uses real workflows and metadata from myExperiment, and compare search results via WorkflowHunt and via myExperimentâs search interface.},<br />
keywords = {Scientific Workflows, Semantic Annotation, Workflow Retrieval}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('446','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_446" style="display:none;"><div class="tp_abstract_entry">Scientific datasets, and the experiments that analyze them are growing in size and complexity, and scientists are facing difficulties to share such resources. Some initiatives have emerged to try to solve this problem. One of them involves the use of scientific workflows to represent and enact experiment execution. There is an increasing number of workflows that are potentially relevant for more than one scientific domain. However, it is hard to find workflows suitable for reuse given an experiment. Creating a workflow takes time and resources, and their reuse helps scientists to build new workflows faster and in a more reliable way. Search mechanisms in workflow repositories should provide different options for workflow discovery, but it is difficult for generic repositories to provide multiple mechanisms. This paper presents WorkflowHunt, a hybrid architecture for workflow search and discovery for generic repositories, which combines keyword and semantic search to allow finding relevant workflows using different search methods. We validated our architecture creating a prototype that uses real workflows and metadata from myExperiment, and compare search results via WorkflowHunt and via myExperimentâs search interface.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('446','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_446" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/PID4958635.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/PID4958635.pdf" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/10/PID4958635.pdf</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('446','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Saraiva,  MÃ¡rcio de Carvalho; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('442','tp_links')" style="cursor:pointer;">Finding out Topics in Educational Materials Using their Components</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Proceedings of THE 47th Annual Frontiers in Education (FIE) Conference, October 18-21, 2017, </span><span class="tp_pub_additional_organization">IEEE </span><span class="tp_pub_additional_address">Indianapolis, Indiana, USA, </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_442" class="tp_show" onclick="teachpress_pub_showhide('442','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_442" class="tp_show" onclick="teachpress_pub_showhide('442','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_442" class="tp_show" onclick="teachpress_pub_showhide('442','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=123#tppubs" title="Show all publications which have a relationship to this tag">Components</a>, <a href="/?page_id=63&amp;tgid=122#tppubs" title="Show all publications which have a relationship to this tag">Content analysis and feature selection</a>, <a href="/?page_id=63&amp;tgid=108#tppubs" title="Show all publications which have a relationship to this tag">educational material</a>, <a href="/?page_id=63&amp;tgid=121#tppubs" title="Show all publications which have a relationship to this tag">Information extraction</a>, <a href="/?page_id=63&amp;tgid=124#tppubs" title="Show all publications which have a relationship to this tag">Topics Classification</a>)</p><div class="tp_bibtex" id="tp_bibtex_442" style="display:none;"><div class="tp_bibtex_entry">@conference{deSaraiva2017,<br />
title = {Finding out Topics in Educational Materials Using their Components},<br />
author = {MÃ¡rcio de Carvalho Saraiva and Claudia Bauzer Medeiros},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/ArtigoFIE2017-MarcioSaraiva.pdf},<br />
year  = {2017},<br />
date = {2017-10-17},<br />
booktitle = {Proceedings of THE 47th Annual Frontiers in Education (FIE) Conference, October 18-21, 2017},<br />
address = {Indianapolis, Indiana, USA},<br />
organization = {IEEE},<br />
abstract = {The Web is witnessing an exponential growth of distributed and heterogeneous educational material. This hampers distinguishing among contents of these materials, as well as their retrieval. While information retrieval and classification mechanisms concentrate on corpus analysis, annotation approaches either target specific formats or require that a document follows interoperable standards. Rather than target only textual characteristics, our strategy is mainly based on components of educational material. The header, body, footer and numbering of slides and progress bar are examples of components of slides and videos. Though our work is general purpose, it is being tested against slides and videos from Coursera, a web platform that provides universal access to online education material and courses from universities and organizations around the world.},<br />
keywords = {Components, Content analysis and feature selection, educational material, Information extraction, Topics Classification}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('442','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_442" style="display:none;"><div class="tp_abstract_entry">The Web is witnessing an exponential growth of distributed and heterogeneous educational material. This hampers distinguishing among contents of these materials, as well as their retrieval. While information retrieval and classification mechanisms concentrate on corpus analysis, annotation approaches either target specific formats or require that a document follows interoperable standards. Rather than target only textual characteristics, our strategy is mainly based on components of educational material. The header, body, footer and numbering of slides and progress bar are examples of components of slides and videos. Though our work is general purpose, it is being tested against slides and videos from Coursera, a web platform that provides universal access to online education material and courses from universities and organizations around the world.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('442','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_442" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/ArtigoFIE2017-MarcioSaraiva.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/ArtigoFIE2017-MarcioSara[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/ArtigoFIE2017-MarcioSara[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('442','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Daltio,  Jaudete</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('448','tp_links')" style="cursor:pointer;">Views over Graph Databases: A Multifocus Approach for Heterogeneous Data</a> <span class="tp_pub_typ">(PhD Thesis)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_school">University of Campinas - Institute of Computing, </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_448" class="tp_show" onclick="teachpress_pub_showhide('448','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_448" class="tp_show" onclick="teachpress_pub_showhide('448','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_448" class="tp_show" onclick="teachpress_pub_showhide('448','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=137#tppubs" title="Show all publications which have a relationship to this tag">graph database; PhDThesis; Views; Multifocus</a>)</p><div class="tp_bibtex" id="tp_bibtex_448" style="display:none;"><div class="tp_bibtex_entry">@phdthesis{Daltio2017,<br />
title = {Views over Graph Databases: A Multifocus Approach for Heterogeneous Data},<br />
author = {Jaudete Daltio},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/12/jaudete_daltio_tese.pdf},<br />
year  = {2017},<br />
date = {2017-09-12},<br />
school = {University of Campinas - Institute of Computing},<br />
abstract = {Scientific research has become data-intensive and data-dependent. This new research paradigm requires sophisticated computer science techniques and technologies to support the life cycle of scientific data and collaboration among scientists from distinct areas. A major requirement is that researchers working in data-intensive interdisciplinary teams demand construction of multiple perspectives of the world, built over the same datasets. Present solutions cover a wide range of aspects, from the design of interoperability standards to the use of non-relational database management systems. None of these efforts, however, adequately meet the needs of multiple perspectives, which are called foci in the thesis. Basically, a focus is designed/built to cater to a research group (even within a single project) that needs to deal with a subset of data of interest, under multiple aggregation/generalization levels. The definition and creation of a focus are complex tasks that require mechanisms and engines to manipulate multiple representations of the same real world phenomenon. This PhD research aims to provide multiple foci over heterogeneous data. To meet this challenge, we deal with four research problems. The first two were (1) choosing an appropriate data management paradigm; and (2) eliciting multifocus requirements. Our work towards solving these problems made as choose graph databases to answer (1) and the concept of views in relational databases for (2). However, there is no consensual data model for graph databases and views are seldom discussed in this context. Thus, research problems (3) and (4) are: (3) specifying an adequate graph data model and (4) defining a framework to handle views on graph databases. Our research in these problems results in the main contributions of this thesis: (i) to present the case for the use of graph databases in multifocus research as persistence layer -- a schemaless and relationship driven type of database that provides a full understanding of data connections; (ii) to define views for graph databases to support the need for multiple foci, considering graph data manipulation, graph algorithms and traversal tasks; (iii) to propose a property graph data model (PGDM) to fill the gap of absence of a full-fledged data model for graphs; (iv) to specify and implement a framework, named Graph-Kaleidoscope, that supports views over graph databases and (v) to validate our framework for real world applications in two domains -- biodiversity and environmental resources -- typical examples of multidisciplinary research that involve the analysis of interactions of phenomena using heterogeneous data.},<br />
keywords = {graph database; PhDThesis; Views; Multifocus}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('448','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_448" style="display:none;"><div class="tp_abstract_entry">Scientific research has become data-intensive and data-dependent. This new research paradigm requires sophisticated computer science techniques and technologies to support the life cycle of scientific data and collaboration among scientists from distinct areas. A major requirement is that researchers working in data-intensive interdisciplinary teams demand construction of multiple perspectives of the world, built over the same datasets. Present solutions cover a wide range of aspects, from the design of interoperability standards to the use of non-relational database management systems. None of these efforts, however, adequately meet the needs of multiple perspectives, which are called foci in the thesis. Basically, a focus is designed/built to cater to a research group (even within a single project) that needs to deal with a subset of data of interest, under multiple aggregation/generalization levels. The definition and creation of a focus are complex tasks that require mechanisms and engines to manipulate multiple representations of the same real world phenomenon. This PhD research aims to provide multiple foci over heterogeneous data. To meet this challenge, we deal with four research problems. The first two were (1) choosing an appropriate data management paradigm; and (2) eliciting multifocus requirements. Our work towards solving these problems made as choose graph databases to answer (1) and the concept of views in relational databases for (2). However, there is no consensual data model for graph databases and views are seldom discussed in this context. Thus, research problems (3) and (4) are: (3) specifying an adequate graph data model and (4) defining a framework to handle views on graph databases. Our research in these problems results in the main contributions of this thesis: (i) to present the case for the use of graph databases in multifocus research as persistence layer -- a schemaless and relationship driven type of database that provides a full understanding of data connections; (ii) to define views for graph databases to support the need for multiple foci, considering graph data manipulation, graph algorithms and traversal tasks; (iii) to propose a property graph data model (PGDM) to fill the gap of absence of a full-fledged data model for graphs; (iv) to specify and implement a framework, named Graph-Kaleidoscope, that supports views over graph databases and (v) to validate our framework for real world applications in two domains -- biodiversity and environmental resources -- typical examples of multidisciplinary research that involve the analysis of interactions of phenomena using heterogeneous data.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('448','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_448" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/12/jaudete_daltio_tese.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/12/jaudete_daltio_tese.pdf" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/12/jaudete_daltio_tese.pdf</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('448','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Carvalho,  Lucas A. M. C.; Malaverri,  Joana E. Gonzales; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('440','tp_links')" style="cursor:pointer;">Implementing W2Share: Supporting Reproducibility and Quality Assessment in eScience</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">11th BreSci - Brazilian e-Science Workshop, </span><span class="tp_pub_additional_publisher">Sociedade Brasileira de ComputaÃ§Ã£o (SBC), </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_440" class="tp_show" onclick="teachpress_pub_showhide('440','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_440" class="tp_show" onclick="teachpress_pub_showhide('440','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_440" class="tp_show" onclick="teachpress_pub_showhide('440','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=31#tppubs" title="Show all publications which have a relationship to this tag">Data quality</a>, <a href="/?page_id=63&amp;tgid=101#tppubs" title="Show all publications which have a relationship to this tag">Provenance Information</a>, <a href="/?page_id=63&amp;tgid=99#tppubs" title="Show all publications which have a relationship to this tag">Scientific Workflows</a>, <a href="/?page_id=63&amp;tgid=25#tppubs" title="Show all publications which have a relationship to this tag">Semantic Annotation</a>, <a href="/?page_id=63&amp;tgid=118#tppubs" title="Show all publications which have a relationship to this tag">W2Share framework</a>)</p><div class="tp_bibtex" id="tp_bibtex_440" style="display:none;"><div class="tp_bibtex_entry">@conference{Carvalho2017,<br />
title = {Implementing W2Share: Supporting Reproducibility and Quality Assessment in eScience},<br />
author = {Lucas A. M. C. Carvalho and Joana E. Gonzales Malaverri and Claudia Bauzer Medeiros},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/05/w2share-bresci2017-camera-ready.pdf},<br />
year  = {2017},<br />
date = {2017-07-06},<br />
booktitle = {11th BreSci - Brazilian e-Science Workshop},<br />
pages = {1-8},<br />
publisher = {Sociedade Brasileira de ComputaÃ§Ã£o (SBC)},<br />
abstract = { An open problem in scientific community is that of supporting reproducibility and quality assessment of scientific experiments. Solutions need to<br />
be able to help scientists to reproduce experimental procedures in a reliable manner and, at the same time, to provide mechanisms for documenting the experiments to enhance integrity and transparency. Moreover, solutions need to incorporate features that allow the assessment of procedures, data used and results of those experiments. In this context, we designed W2Share, a framework to meet these requirements. This paper introduces our first implementation of W2Share, which moreover guides scientists in step-by-step process to ensure reproducibility based on a script-to-workflow conversion strategy. W2Share also incorporates features that allow annotating experiments with quality information. We validate our prototype using a real-world scenario in Bioinformatics.},<br />
keywords = {Data quality, Provenance Information, Scientific Workflows, Semantic Annotation, W2Share framework}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('440','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_440" style="display:none;"><div class="tp_abstract_entry"> An open problem in scientific community is that of supporting reproducibility and quality assessment of scientific experiments. Solutions need to<br />
be able to help scientists to reproduce experimental procedures in a reliable manner and, at the same time, to provide mechanisms for documenting the experiments to enhance integrity and transparency. Moreover, solutions need to incorporate features that allow the assessment of procedures, data used and results of those experiments. In this context, we designed W2Share, a framework to meet these requirements. This paper introduces our first implementation of W2Share, which moreover guides scientists in step-by-step process to ensure reproducibility based on a script-to-workflow conversion strategy. W2Share also incorporates features that allow annotating experiments with quality information. We validate our prototype using a real-world scenario in Bioinformatics.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('440','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_440" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/05/w2share-bresci2017-camera-ready.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/05/w2share-bresci2017-camer[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/05/w2share-bresci2017-camer[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('440','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Tacioli,  Leandro; Toledo,  LuÃ­s Felipe; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('441','tp_links')" style="cursor:pointer;">An Architecture for Animal Sound Identification based on Multiple Feature Extraction and Classification Algorithms</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">11th BreSci - Brazilian e-Science Workshop, </span><span class="tp_pub_additional_publisher">Sociedade Brasileira de ComputaÃ§Ã£o (SBC), </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_441" class="tp_show" onclick="teachpress_pub_showhide('441','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_441" class="tp_show" onclick="teachpress_pub_showhide('441','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_441" class="tp_show" onclick="teachpress_pub_showhide('441','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=119#tppubs" title="Show all publications which have a relationship to this tag">eScience</a>, <a href="/?page_id=63&amp;tgid=120#tppubs" title="Show all publications which have a relationship to this tag">Feature Extraction</a>, <a href="/?page_id=63&amp;tgid=82#tppubs" title="Show all publications which have a relationship to this tag">Pattern recognition</a>)</p><div class="tp_bibtex" id="tp_bibtex_441" style="display:none;"><div class="tp_bibtex_entry">@conference{Tacioli2017,<br />
title = {An Architecture for Animal Sound Identification based on Multiple Feature Extraction and Classification Algorithms},<br />
author = {Leandro Tacioli and LuÃ­s Felipe Toledo and Claudia Bauzer Medeiros},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/06/Tacioli-BreSci2017.pdf},<br />
year  = {2017},<br />
date = {2017-07-06},<br />
booktitle = {11th BreSci - Brazilian e-Science Workshop},<br />
publisher = {Sociedade Brasileira de ComputaÃ§Ã£o (SBC)},<br />
abstract = {Automatic identification of animals is extremely useful for scientists, providing ways to monitor species and changes in ecological communities. The choice of effective audio features and classification techniques is a challenge on any audio recognition system, especially in bioacoustics that commonly uses several algorithms. This paper presents a novel software architecture that supports multiple feature extraction and classification algorithms to help on the identification of animal species from their recorded sounds. This architecture was implemented by the WASIS software, freely available on the Web.},<br />
keywords = {eScience, Feature Extraction, Pattern recognition}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('441','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_441" style="display:none;"><div class="tp_abstract_entry">Automatic identification of animals is extremely useful for scientists, providing ways to monitor species and changes in ecological communities. The choice of effective audio features and classification techniques is a challenge on any audio recognition system, especially in bioacoustics that commonly uses several algorithms. This paper presents a novel software architecture that supports multiple feature extraction and classification algorithms to help on the identification of animal species from their recorded sounds. This architecture was implemented by the WASIS software, freely available on the Web.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('441','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_441" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/06/Tacioli-BreSci2017.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/06/Tacioli-BreSci2017.pdf" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/06/Tacioli-BreSci2017.pdf</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('441','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Tacioli,  Leandro</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('443','tp_links')" style="cursor:pointer;">WASIS - Bioacoustic Species Identification based on Multiple Feature Extraction and Classification Algorithms</a> <span class="tp_pub_typ">(Mastersthesis)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_school">Universidade Estadual de Campinas - UNICAMP, </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_443" class="tp_show" onclick="teachpress_pub_showhide('443','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_443" class="tp_show" onclick="teachpress_pub_showhide('443','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_443" class="tp_show" onclick="teachpress_pub_showhide('443','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=127#tppubs" title="Show all publications which have a relationship to this tag">Animals - Identification</a>, <a href="/?page_id=63&amp;tgid=126#tppubs" title="Show all publications which have a relationship to this tag">Bioacoustics</a>, <a href="/?page_id=63&amp;tgid=125#tppubs" title="Show all publications which have a relationship to this tag">Computer systems</a>, <a href="/?page_id=63&amp;tgid=82#tppubs" title="Show all publications which have a relationship to this tag">Pattern recognition</a>)</p><div class="tp_bibtex" id="tp_bibtex_443" style="display:none;"><div class="tp_bibtex_entry">@mastersthesis{Tacioli2017b,<br />
title = {WASIS - Bioacoustic Species Identification based on Multiple Feature Extraction and Classification Algorithms},<br />
author = {Leandro Tacioli},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/LeandroTacioli-Mestrado.pdf},<br />
year  = {2017},<br />
date = {2017-07-03},<br />
school = {Universidade Estadual de Campinas - UNICAMP},<br />
abstract = {Automatic identification of animal species based on their sounds is one of the means to conduct research in bioacoustics. This research domain provides, for instance, ways to monitor rare and endangered species, to analyze changes in ecological communities, or ways to study the social meaning of animal calls in their behavioral contexts. Identification mechanisms are typically executed in two stages: feature extraction and classification. Both stages present challenges, in computer science and in bioacoustics. The choice of effective feature extraction and classification algorithms is a challenge on any audio recognition system, especially in bioacoustics. Considering the wide variety of animal groups studied, algorithms are tailored to specific groups. Audio classification techniques are also sensitive to the extracted features, and conditions surrounding the recordings. As a results, most bioacoustic softwares are not extensible, therefore limiting the kinds of recognition experiments that can be conducted. Given this scenario, this dissertation proposes a software architecture that allows multiple feature extraction, feature fusion and classification algorithms to support scientists and the general public on the identification of animal species through their recorded sounds. This architecture was implemented by the WASIS software, freely available on the Web. Since WASIS is open-source and expansible, experts can perform experiments with many combinations of pairs descriptor-classifier to choose the most appropriate ones for the identification of given animal sub-groups. A number of algorithms were implemented, serving as the basis for a comparative study that recommends sets of feature extraction and classification algorithms for three animal groups.},<br />
keywords = {Animals - Identification, Bioacoustics, Computer systems, Pattern recognition}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('443','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_443" style="display:none;"><div class="tp_abstract_entry">Automatic identification of animal species based on their sounds is one of the means to conduct research in bioacoustics. This research domain provides, for instance, ways to monitor rare and endangered species, to analyze changes in ecological communities, or ways to study the social meaning of animal calls in their behavioral contexts. Identification mechanisms are typically executed in two stages: feature extraction and classification. Both stages present challenges, in computer science and in bioacoustics. The choice of effective feature extraction and classification algorithms is a challenge on any audio recognition system, especially in bioacoustics. Considering the wide variety of animal groups studied, algorithms are tailored to specific groups. Audio classification techniques are also sensitive to the extracted features, and conditions surrounding the recordings. As a results, most bioacoustic softwares are not extensible, therefore limiting the kinds of recognition experiments that can be conducted. Given this scenario, this dissertation proposes a software architecture that allows multiple feature extraction, feature fusion and classification algorithms to support scientists and the general public on the identification of animal species through their recorded sounds. This architecture was implemented by the WASIS software, freely available on the Web. Since WASIS is open-source and expansible, experts can perform experiments with many combinations of pairs descriptor-classifier to choose the most appropriate ones for the identification of given animal sub-groups. A number of algorithms were implemented, serving as the basis for a comparative study that recommends sets of feature extraction and classification algorithms for three animal groups.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('443','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_443" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/LeandroTacioli-Mestrado.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/LeandroTacioli-Mestrado.[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/07/LeandroTacioli-Mestrado.[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('443','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Filho,  Francisco JosÃ© Nardi</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('444','tp_links')" style="cursor:pointer;">Hybrid Narrative and Clinical Knowledge Base for Emergency Medicine Training</a> <span class="tp_pub_typ">(Mastersthesis)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_school">Universidade Estadual de Campinas - UNICAMP, </span><span class="tp_pub_additional_year">2017</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_444" class="tp_show" onclick="teachpress_pub_showhide('444','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_444" class="tp_show" onclick="teachpress_pub_showhide('444','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_444" class="tp_show" onclick="teachpress_pub_showhide('444','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=132#tppubs" title="Show all publications which have a relationship to this tag">Artificial intelligence - Medical applications</a>, <a href="/?page_id=63&amp;tgid=131#tppubs" title="Show all publications which have a relationship to this tag">Emergency medicine</a>, <a href="/?page_id=63&amp;tgid=129#tppubs" title="Show all publications which have a relationship to this tag">Expert systems (Computer science)</a>, <a href="/?page_id=63&amp;tgid=130#tppubs" title="Show all publications which have a relationship to this tag">Information storage and retrieval systems</a>, <a href="/?page_id=63&amp;tgid=128#tppubs" title="Show all publications which have a relationship to this tag">Information systems</a>)</p><div class="tp_bibtex" id="tp_bibtex_444" style="display:none;"><div class="tp_bibtex_entry">@mastersthesis{,<br />
title = {Hybrid Narrative and Clinical Knowledge Base for Emergency Medicine Training},<br />
author = {Francisco JosÃ© Nardi Filho},<br />
url = {http://www.repositorio.unicamp.br/bitstream/REPOSIP/325067/1/Nardi%20Filho_FranciscoJose_M.pdf},<br />
year  = {2017},<br />
date = {2017-05-19},<br />
school = {Universidade Estadual de Campinas - UNICAMP},<br />
abstract = {Software for medical training usually follows two types of approaches for the representation of its data. One type is the software of simulation-based training or virtual patients - which have highly structured representations of the clinical data and simulation plans. Another type is some systems which focus on the narrative of a clinical case in free-text format - e.g., the Jacinto emergency medicine learning environment. In this case, the clinical data mixes with the narrative in unstructured format. Thus, we propose a model for a hybrid narrative and clinical knowledge base for emergency medicine training that combines both of the approaches. We hypothesize that by connecting narratives with structured clinical information, we can take advantage of the strongest points of each approach. On the one hand, structured clinical data offers flexibility for the production of case variations and alternative plans, which gives machine more autonomy to assess user performance. On the other hand, free-text narratives enable the introduction of real scenario relevant aspects and context, beyond clinical data. In this work, we present a practical experiment involving the database of the Jacinto emergency medicine learning environment.},<br />
keywords = {Artificial intelligence - Medical applications, Emergency medicine, Expert systems (Computer science), Information storage and retrieval systems, Information systems}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('444','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_444" style="display:none;"><div class="tp_abstract_entry">Software for medical training usually follows two types of approaches for the representation of its data. One type is the software of simulation-based training or virtual patients - which have highly structured representations of the clinical data and simulation plans. Another type is some systems which focus on the narrative of a clinical case in free-text format - e.g., the Jacinto emergency medicine learning environment. In this case, the clinical data mixes with the narrative in unstructured format. Thus, we propose a model for a hybrid narrative and clinical knowledge base for emergency medicine training that combines both of the approaches. We hypothesize that by connecting narratives with structured clinical information, we can take advantage of the strongest points of each approach. On the one hand, structured clinical data offers flexibility for the production of case variations and alternative plans, which gives machine more autonomy to assess user performance. On the other hand, free-text narratives enable the introduction of real scenario relevant aspects and context, beyond clinical data. In this work, we present a practical experiment involving the database of the Jacinto emergency medicine learning environment.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('444','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_444" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.repositorio.unicamp.br/bitstream/REPOSIP/325067/1/Nardi%20Filho_FranciscoJose_M.pdf" title="http://www.repositorio.unicamp.br/bitstream/REPOSIP/325067/1/Nardi%20Filho_Franc[...]" target="_blank">http://www.repositorio.unicamp.br/bitstream/REPOSIP/325067/1/Nardi%20Filho_Franc[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('444','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr><td><h3 class="tp_h3">2016</h3></td></tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Pantoja,  Fagner L.; Cavoto,  PatrÃ­cia; Reis,  Julio Cesar dos; SantanchÃ¨,  AndrÃ©</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('437','tp_links')" style="cursor:pointer;">Generating Knowledge Networks from Phenotypic Descriptions</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Proceedings of the 12th International Conference on eScience, </span><span class="tp_pub_additional_organization">IEEE, </span><span class="tp_pub_additional_address">Baltimore, MD, USA, </span><span class="tp_pub_additional_year">2016</span>.</p><p class="tp_pub_tags">(<a id="tp_links_sh_437" class="tp_show" onclick="teachpress_pub_showhide('437','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_437" class="tp_show" onclick="teachpress_pub_showhide('437','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=110#tppubs" title="Show all publications which have a relationship to this tag">Curation</a>)</p><div class="tp_bibtex" id="tp_bibtex_437" style="display:none;"><div class="tp_bibtex_entry">@conference{@inproceedings{Pantoja,author={Pantoja,FagnerLeala,<br />
title = {Generating Knowledge Networks from Phenotypic Descriptions},<br />
author = {Fagner L. Pantoja and PatrÃ­cia Cavoto and Julio Cesar dos Reis and AndrÃ© SantanchÃ¨},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/1095anav.pdf},<br />
year  = {2016},<br />
date = {2016-10-24},<br />
booktitle = {Proceedings of the 12th International Conference on eScience},<br />
address = {Baltimore, MD, USA},<br />
organization = {IEEE,},<br />
keywords = {Curation}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('437','tp_bibtex')">Close</a></p></div><div class="tp_links" id="tp_links_437" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/1095anav.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/1095anav.pdf" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/1095anav.pdf</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('437','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Carvalho,  Lucas A. M. C.; Belhajjame,  Khalid; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('431','tp_links')" style="cursor:pointer;">Converting Scripts into Reproducible Workflow Research Objects</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Proceedings of the 2016 IEEE 12th International Conference on eScience, </span><span class="tp_pub_additional_publisher">IEEE, </span><span class="tp_pub_additional_address">Baltimore, MD, USA, October 23-27, </span><span class="tp_pub_additional_year">2016</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_431" class="tp_show" onclick="teachpress_pub_showhide('431','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_431" class="tp_show" onclick="teachpress_pub_showhide('431','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_431" class="tp_show" onclick="teachpress_pub_showhide('431','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=104#tppubs" title="Show all publications which have a relationship to this tag">Methodology</a>, <a href="/?page_id=63&amp;tgid=101#tppubs" title="Show all publications which have a relationship to this tag">Provenance Information</a>, <a href="/?page_id=63&amp;tgid=99#tppubs" title="Show all publications which have a relationship to this tag">Scientific Workflows</a>)</p><div class="tp_bibtex" id="tp_bibtex_431" style="display:none;"><div class="tp_bibtex_entry">@conference{Carvalho2016Converting,<br />
title = {Converting Scripts into Reproducible Workflow Research Objects},<br />
author = {Lucas A. M. C. Carvalho and Khalid Belhajjame and Claudia Bauzer Medeiros},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2016/08/converting-scripts-reproducible-camera-ready.pdf},<br />
year  = {2016},<br />
date = {2016-10-23},<br />
booktitle = {Proceedings of the 2016 IEEE 12th International Conference on eScience},<br />
publisher = {IEEE},<br />
address = {Baltimore, MD, USA, October 23-27},<br />
abstract = {Scientific discovery and analysis are increasingly computational and data-driven. While scripting languages, such as Python, R and Perl, are the means of choice of the majority of scientists to encode and run their data analysis, scripts are generally not amenable to reuse or reproducibility. Scripts do rarely get reused or even shared with third party scientists. We argue in this paper that the reproducibility of scripts can be promoted by converting them into workflow research objects. A workflow research object encodes a script into a production (executable) workflow that is accompanied by annotations, example datasets and provenance traces of their execution, thereby allowing third party users to understand the data analysis encoded by the original script, run the associated workflow using the same or different dataset, or even repurpose it for a different analysis. To this end, we present a methodology for converting scripts into workflow research objects in a principled manner, guided by requirements that we elicited for this purpose. The methodology  exploits tools and standards that have been developed by the community, in particular YesWorkflow, Research Objects and the W3C PROV. It is showcased using a real world use case from the field of Molecular Dynamics.  },<br />
keywords = {Methodology, Provenance Information, Scientific Workflows}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('431','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_431" style="display:none;"><div class="tp_abstract_entry">Scientific discovery and analysis are increasingly computational and data-driven. While scripting languages, such as Python, R and Perl, are the means of choice of the majority of scientists to encode and run their data analysis, scripts are generally not amenable to reuse or reproducibility. Scripts do rarely get reused or even shared with third party scientists. We argue in this paper that the reproducibility of scripts can be promoted by converting them into workflow research objects. A workflow research object encodes a script into a production (executable) workflow that is accompanied by annotations, example datasets and provenance traces of their execution, thereby allowing third party users to understand the data analysis encoded by the original script, run the associated workflow using the same or different dataset, or even repurpose it for a different analysis. To this end, we present a methodology for converting scripts into workflow research objects in a principled manner, guided by requirements that we elicited for this purpose. The methodology  exploits tools and standards that have been developed by the community, in particular YesWorkflow, Research Objects and the W3C PROV. It is showcased using a real world use case from the field of Molecular Dynamics.  </div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('431','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_431" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/08/converting-scripts-reproducible-camera-ready.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/08/converting-scripts-repro[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2016/08/converting-scripts-repro[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('431','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Carvalho,  Lucas A. M. C.; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('432','tp_links')" style="cursor:pointer;">Provenance-Based Infrastructure to Support Reuse of Computational Experiments</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Proceedings of the Satellite Events of the 31st Brazilian Symposium on Databases (Thesis and Dissertations Workshop), </span><span class="tp_pub_additional_organization">Sociedade Brasileira de ComputaÃ§Ã£o </span><span class="tp_pub_additional_address">Salvador, Bahia, Brazil, October 4-7, </span><span class="tp_pub_additional_year">2016</span>, <span class="tp_pub_additional_isbn">ISBN: 978-85-7669-343-7</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_432" class="tp_show" onclick="teachpress_pub_showhide('432','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_432" class="tp_show" onclick="teachpress_pub_showhide('432','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_432" class="tp_show" onclick="teachpress_pub_showhide('432','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=101#tppubs" title="Show all publications which have a relationship to this tag">Provenance Information</a>, <a href="/?page_id=63&amp;tgid=99#tppubs" title="Show all publications which have a relationship to this tag">Scientific Workflows</a>, <a href="/?page_id=63&amp;tgid=25#tppubs" title="Show all publications which have a relationship to this tag">Semantic Annotation</a>, <a href="/?page_id=63&amp;tgid=100#tppubs" title="Show all publications which have a relationship to this tag">Workflow Retrieval</a>)</p><div class="tp_bibtex" id="tp_bibtex_432" style="display:none;"><div class="tp_bibtex_entry">@conference{Carvalho2016Wtd,<br />
title = {Provenance-Based Infrastructure to Support Reuse of Computational Experiments},<br />
author = {Lucas A. M. C. Carvalho and Claudia Bauzer Medeiros},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/wtdbd2016-camera-ready.pdf},<br />
isbn = {978-85-7669-343-7},<br />
year  = {2016},<br />
date = {2016-10-04},<br />
booktitle = {Proceedings of the Satellite Events of the 31st Brazilian Symposium on Databases (Thesis and Dissertations Workshop)},<br />
pages = {74-81},<br />
address = {Salvador, Bahia, Brazil, October 4-7},<br />
organization = {Sociedade Brasileira de ComputaÃ§Ã£o},<br />
abstract = {One recurrent problem in multidisciplinary research is finding reusable objects (e.g., scripts, code, documents, workflows) that can be used across disciplines to enhance collaboration.<br />
This paper presents our ongoing work taking advantage of provenance information, combined with scientific workflows, to help find such objects. We also present challenges posed by provenance-based retrieval, which we propose as a solution for transdisciplinary scientific collaboration via reuse.<br />
Our case study in molecular dynamics experiments is part of a larger multi-scale experimental scenario that requires cooperation involving scientists from different disciplines.},<br />
keywords = {Provenance Information, Scientific Workflows, Semantic Annotation, Workflow Retrieval}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('432','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_432" style="display:none;"><div class="tp_abstract_entry">One recurrent problem in multidisciplinary research is finding reusable objects (e.g., scripts, code, documents, workflows) that can be used across disciplines to enhance collaboration.<br />
This paper presents our ongoing work taking advantage of provenance information, combined with scientific workflows, to help find such objects. We also present challenges posed by provenance-based retrieval, which we propose as a solution for transdisciplinary scientific collaboration via reuse.<br />
Our case study in molecular dynamics experiments is part of a larger multi-scale experimental scenario that requires cooperation involving scientists from different disciplines.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('432','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_432" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/wtdbd2016-camera-ready.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/wtdbd2016-camera-ready.p[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/wtdbd2016-camera-ready.p[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('432','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Saraiva,  MÃ¡rcio de Carvalho; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('434','tp_links')" style="cursor:pointer;">Use of graphs and taxonomic classifications to analyze content relationships among courseware</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Proceedings of the 31st Brazilian Symposium on Databases, </span><span class="tp_pub_additional_organization">Sociedade Brasileira de ComputaÃ§Ã£o </span><span class="tp_pub_additional_address">Salvador, Bahia, Brazil, October 4-7, </span><span class="tp_pub_additional_year">2016</span>, <span class="tp_pub_additional_issn">ISSN: 2316-5170</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_434" class="tp_show" onclick="teachpress_pub_showhide('434','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_434" class="tp_show" onclick="teachpress_pub_showhide('434','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_434" class="tp_show" onclick="teachpress_pub_showhide('434','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=107#tppubs" title="Show all publications which have a relationship to this tag">content analysis</a>, <a href="/?page_id=63&amp;tgid=108#tppubs" title="Show all publications which have a relationship to this tag">educational material</a>, <a href="/?page_id=63&amp;tgid=70#tppubs" title="Show all publications which have a relationship to this tag">graph database</a>, <a href="/?page_id=63&amp;tgid=109#tppubs" title="Show all publications which have a relationship to this tag">multiple relationships</a>)</p><div class="tp_bibtex" id="tp_bibtex_434" style="display:none;"><div class="tp_bibtex_entry">@conference{Saraiva2016Short,<br />
title = {Use of graphs and taxonomic classifications to analyze content relationships among courseware},<br />
author = {MÃ¡rcio de Carvalho Saraiva and Claudia Bauzer Medeiros},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/marciosaraiva-sbbd2016.pdf},<br />
issn = {2316-5170},<br />
year  = {2016},<br />
date = {2016-10-04},<br />
booktitle = {Proceedings of the 31st Brazilian Symposium on Databases},<br />
pages = {265-270},<br />
address = {Salvador, Bahia, Brazil, October 4-7},<br />
organization = {Sociedade Brasileira de ComputaÃ§Ã£o},<br />
abstract = {The search for educational content in courseware repositories is  laborious and time consuming. There is an abundance of such repositories, and research efforts to facilitate search, but access is guided by keywords and/or terms selected by courseware authors, thus lacking flexibility. The goal of this project is to design and develop a suite of tools to assist users to find, analyze and select pieces of educational content that are relevant to their learning goals. Contributions will be both at the algorithm and software design level, and at the user (application) level.},<br />
keywords = {content analysis, educational material, graph database, multiple relationships}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('434','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_434" style="display:none;"><div class="tp_abstract_entry">The search for educational content in courseware repositories is  laborious and time consuming. There is an abundance of such repositories, and research efforts to facilitate search, but access is guided by keywords and/or terms selected by courseware authors, thus lacking flexibility. The goal of this project is to design and develop a suite of tools to assist users to find, analyze and select pieces of educational content that are relevant to their learning goals. Contributions will be both at the algorithm and software design level, and at the user (application) level.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('434','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_434" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/marciosaraiva-sbbd2016.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/marciosaraiva-sbbd2016.p[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2017/01/marciosaraiva-sbbd2016.p[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('434','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Borges,  Luana Loubet</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('436','tp_links')" style="cursor:pointer;">BioGraph: Linking Biological Bases Across Organisms</a> <span class="tp_pub_typ">(Mastersthesis)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_school">Universidade Estadual de Campinas - UNICAMP, </span><span class="tp_pub_additional_year">2016</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_436" class="tp_show" onclick="teachpress_pub_showhide('436','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_436" class="tp_show" onclick="teachpress_pub_showhide('436','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_436" class="tp_show" onclick="teachpress_pub_showhide('436','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=5#tppubs" title="Show all publications which have a relationship to this tag">database</a>, <a href="/?page_id=63&amp;tgid=103#tppubs" title="Show all publications which have a relationship to this tag">Ontologies (Information retrieval)</a>)</p><div class="tp_bibtex" id="tp_bibtex_436" style="display:none;"><div class="tp_bibtex_entry">@mastersthesis{Borges2016,<br />
title = {BioGraph: Linking Biological Bases Across Organisms},<br />
author = {Luana Loubet Borges},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2016/10/dissertacao_versao_final_Luana_Loubet.pdf},<br />
year  = {2016},<br />
date = {2016-08-05},<br />
school = {Universidade Estadual de Campinas - UNICAMP},<br />
abstract = {Representing data as networks have been shown to be a powerful approach for data analysis in biodiversity, e.g., interactions among organisms; relations among genes and phenotypes etc. In this context, databases and repositories following a graph model (e.g., RDF) have been increasingly used to interconnect information and to support network-driven analysis. Usually, this kind of analysis requires gathering together and linking data from several distinct and heterogeneous sources. In this work, we investigate this challenge in the context of biological bases focusing on the characterization of living organisms, especially their phenotypes and diseases. It includes the rich diversity of Model Organism Databases (MODs) -- repositories specialized in a particular taxon -- widely used in the biological and medical studies. We exploit a lightweight integration approach, inspired in the Linked Open Data initiative, mapping several biological bases in a unified graph database -- our BioGraph -- and linking key elements to offer an interconnected view over the data. We present here practical experiments to validate the proposal and to show how BioGraph can contribute for biological data analysis in a network perspective.},<br />
keywords = {database, Ontologies (Information retrieval)}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('436','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_436" style="display:none;"><div class="tp_abstract_entry">Representing data as networks have been shown to be a powerful approach for data analysis in biodiversity, e.g., interactions among organisms; relations among genes and phenotypes etc. In this context, databases and repositories following a graph model (e.g., RDF) have been increasingly used to interconnect information and to support network-driven analysis. Usually, this kind of analysis requires gathering together and linking data from several distinct and heterogeneous sources. In this work, we investigate this challenge in the context of biological bases focusing on the characterization of living organisms, especially their phenotypes and diseases. It includes the rich diversity of Model Organism Databases (MODs) -- repositories specialized in a particular taxon -- widely used in the biological and medical studies. We exploit a lightweight integration approach, inspired in the Linked Open Data initiative, mapping several biological bases in a unified graph database -- our BioGraph -- and linking key elements to offer an interconnected view over the data. We present here practical experiments to validate the proposal and to show how BioGraph can contribute for biological data analysis in a network perspective.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('436','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_436" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/10/dissertacao_versao_final_Luana_Loubet.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/10/dissertacao_versao_final[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2016/10/dissertacao_versao_final[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('436','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Pantoja,  Fagner Leal</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('438','tp_links')" style="cursor:pointer;">Generating Knowledge Networks from Phenotypic Descriptions</a> <span class="tp_pub_typ">(Mastersthesis)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_school">University of Campinas, </span><span class="tp_pub_additional_year">2016</span>.</p><p class="tp_pub_tags">(<a id="tp_links_sh_438" class="tp_show" onclick="teachpress_pub_showhide('438','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_438" class="tp_show" onclick="teachpress_pub_showhide('438','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=110#tppubs" title="Show all publications which have a relationship to this tag">Curation</a>)</p><div class="tp_bibtex" id="tp_bibtex_438" style="display:none;"><div class="tp_bibtex_entry">@mastersthesis{@MastersThesis{Pantoja2016,author={Pantoja,Fagner},<br />
title = {Generating Knowledge Networks from Phenotypic Descriptions},<br />
author = {Fagner Leal Pantoja},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/fagner-dissertacao-vfinal.pdf},<br />
year  = {2016},<br />
date = {2016-08-05},<br />
school = {University of Campinas},<br />
keywords = {Curation}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('438','tp_bibtex')">Close</a></p></div><div class="tp_links" id="tp_links_438" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/fagner-dissertacao-vfinal.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/fagner-dissertacao-vfina[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2016/12/fagner-dissertacao-vfina[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('438','tp_links')">Close</a></p></div>
                </td>
                
                </tr><tr class="tp_publication"><td class="tp_pub_info"><p class="tp_pub_author">Carvalho,  Lucas A. M. C.; Silveira,  Rodrigo L.; Pereira,  Caroline S.; Skaf,  Munir S.; Medeiros,  Claudia Bauzer</p><p class="tp_pub_title"><a class="tp_title_link" onclick="teachpress_pub_showhide('429','tp_links')" style="cursor:pointer;">Provenance-Based Retrieval: Fostering Reuse and Reproducibility Across Scientific Disciplines</a> <span class="tp_pub_typ">(Conference)</span></p><p class="tp_pub_additional"><span class="tp_pub_additional_booktitle">Provenance and Annotation of Data and Processes (Proceedings of 6th International Provenance and Annotation Workshop - IPAW 2016), </span><span class="tp_pub_additional_series">LNCS 9672 </span><span class="tp_pub_additional_publisher">Springer International Publishing, </span><span class="tp_pub_additional_address">McLean, Virginia, U.S.A., </span><span class="tp_pub_additional_year">2016</span>, <span class="tp_pub_additional_isbn">ISBN: 978-3319405926</span>.</p><p class="tp_pub_tags">(<a id="tp_abstract_sh_429" class="tp_show" onclick="teachpress_pub_showhide('429','tp_abstract')" title="Show abstract" style="cursor:pointer;">Abstract</a> | <a id="tp_links_sh_429" class="tp_show" onclick="teachpress_pub_showhide('429','tp_links')" title="Show links and resources" style="cursor:pointer;">Links</a> | <a id="tp_bibtex_sh_429" class="tp_show" onclick="teachpress_pub_showhide('429','tp_bibtex')" style="cursor:pointer;" title="Show BibTeX entry">BibTeX</a> | Tags: <a href="/?page_id=63&amp;tgid=101#tppubs" title="Show all publications which have a relationship to this tag">Provenance Information</a>, <a href="/?page_id=63&amp;tgid=99#tppubs" title="Show all publications which have a relationship to this tag">Scientific Workflows</a>, <a href="/?page_id=63&amp;tgid=25#tppubs" title="Show all publications which have a relationship to this tag">Semantic Annotation</a>, <a href="/?page_id=63&amp;tgid=100#tppubs" title="Show all publications which have a relationship to this tag">Workflow Retrieval</a>)</p><div class="tp_bibtex" id="tp_bibtex_429" style="display:none;"><div class="tp_bibtex_entry">@conference{Carvalho2016,<br />
title = {Provenance-Based Retrieval: Fostering Reuse and Reproducibility Across Scientific Disciplines},<br />
author = {Lucas A. M. C. Carvalho and Rodrigo L. Silveira and Caroline S. Pereira and Munir S. Skaf and Claudia Bauzer Medeiros},<br />
editor = {Marta Mattoso and Boris Glavic},<br />
url = {http://www.lis.ic.unicamp.br/wp-content/uploads/2016/05/ipaw2016-poster-cameraready.pdf},<br />
isbn = {978-3319405926},<br />
year  = {2016},<br />
date = {2016-06-06},<br />
booktitle = {Provenance and Annotation of Data and Processes (Proceedings of 6th International Provenance and Annotation Workshop - IPAW 2016)},<br />
pages = {1-4},<br />
publisher = {Springer International Publishing},<br />
address = {McLean, Virginia, U.S.A.},<br />
chapter = {17},<br />
series = {LNCS 9672},<br />
abstract = {When computational researchers from several domains cooperate, one recurrent problem is finding tools,<br />
methods and approaches that can be used across disciplines, to enhance collaboration through reuse. The<br />
paper presents our ongoing work to meet the challenges posed by provenance-based retrieval, proposed as<br />
a solution for transdisciplinary scientific collaboration via reuse of scientific workflows. Our work is based<br />
upon a case study in molecular dynamics experiments, as part of a larger multi-scale experimental<br />
scenario.},<br />
keywords = {Provenance Information, Scientific Workflows, Semantic Annotation, Workflow Retrieval}<br />
}<br />
</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('429','tp_bibtex')">Close</a></p></div><div class="tp_abstract" id="tp_abstract_429" style="display:none;"><div class="tp_abstract_entry">When computational researchers from several domains cooperate, one recurrent problem is finding tools,<br />
methods and approaches that can be used across disciplines, to enhance collaboration through reuse. The<br />
paper presents our ongoing work to meet the challenges posed by provenance-based retrieval, proposed as<br />
a solution for transdisciplinary scientific collaboration via reuse of scientific workflows. Our work is based<br />
upon a case study in molecular dynamics experiments, as part of a larger multi-scale experimental<br />
scenario.</div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('429','tp_abstract')">Close</a></p></div><div class="tp_links" id="tp_links_429" style="display:none;"><div class="tp_links_entry"><ul class="tp_pub_list"><li><a class="tp_pub_list" style="background-image: url(/wp-content/plugins/teachpress/images/mimetypes/application-pdf.png)" href="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/05/ipaw2016-poster-cameraready.pdf" title="http://www.lis.ic.unicamp.br/wp-content/uploads/2016/05/ipaw2016-poster-camerare[...]" target="_blank">http://www.lis.ic.unicamp.br/wp-content/uploads/2016/05/ipaw2016-poster-camerare[...]</a></li></ul></div><p class="tp_close_menu"><a class="tp_close" onclick="teachpress_pub_showhide('429','tp_links')">Close</a></p></div>
                </td>
                
                </tr></table><div class="tablenav"><div class="tablenav-pages"><span class="displaying-num">440 entries</span> <a class="first-page disabled">&laquo;</a> <a class="prev-page disabled">&lsaquo;</a>  1 of 28 <a href="/?page_id=63&amp;limit=2&amp;" title="next page" class="page-numbers">&rsaquo;</a> <a href="/?page_id=63&amp;limit=28&amp;" title="last page" class="page-numbers">&raquo;</a> </div></div></div>
					
